.eslintrc.js

```diff
@@ -0,0 +1,94 @@
+module.exports = {
```

Comment:
What would be super helpful to me is if you could let me know which files were auto-generated, and where you've made app-specific changes.

Reply:
Hmmm I'm not totally sure how to do that. I did use a codemod to migrate from chai to jest, and I used the `lint --fix` option to auto fix some issues. Let me write a comment to try and guide you through the changes a bit

Reply:
thank you!  the comment is super useful too!